src/Mvc/Mvc.RazorPages/ref/Microsoft.AspNetCore.Mvc.RazorPages.netcoreapp3.0.cs

```diff
@@ -368,6 +368,7 @@ public abstract partial class PageBase : Microsoft.AspNetCore.Mvc.Razor.RazorPag
     {
         protected PageBase() { }
         public Microsoft.AspNetCore.Http.HttpContext HttpContext { get { throw null; } }
+        public Microsoft.AspNetCore.Mvc.ModelBinding.IModelMetadataProvider MetadataProvider { get { throw null; } set { } }
```

Comment:
We don't add API in patch releases.

Reply:
Ahh, didn't realize it. I'll file a thing to make this public. It's non-trivial to test this without the public setter.

Reply:
OK, if there's a good reason let's just go ahead it with.